src/pages/SSLModule/list/index.tsx

```diff
@@ -43,18 +43,29 @@ const List: React.FC = () => {
       title: 'SNI',
       dataIndex: ['value', 'sni'],
     },
+    // TODO: need to check api response
+    {
+      title: '关联路由',
+      dataIndex: [],
+    },
+    {
+      title: '过期时间',
+      dataIndex: [],
+    },
+    {
+      title: '是否启用',
+      valueType: 'option',
+      render: (_, record) => (
+        <>
+          <Switch defaultChecked onClick={() => console.log(record)} />
```

Comment:
Please be careful with the defaultChecked, it should be checked by API response.